flow/src/main/java/com/sequenceiq/flow/core/FlowRegister.java

```diff
@@ -22,13 +23,17 @@
     @Inject
     private MetricService metricService;
 
-    private final Map<String, Pair<Flow, String>> runningFlows = new ConcurrentHashMap<>();
+    private Map<String, Pair<Flow, String>> runningFlows = new ConcurrentHashMap<>();
```

Comment:
Could be assigned null here since this ConcurrentHashMap will be thrown to trash in the @PostConstruct method.

Ideal solution would be initialisation in the constructor and not using PostConstruct, but such things cannot be suggested in this project :D